test-utils/src/main/java/io/littlehorse/test/LHExtension.java

```diff
@@ -226,18 +228,26 @@ private void maybeCreateTenantAndPrincipal(TestContext testContext) {
         if (testContext.getConfig().getTenantId() == null) {
             return;
         }
-        Awaitility.await()
-                .atMost(Duration.ofSeconds(25))
-                .ignoreExceptionsMatching(exn -> RuntimeException.class.isAssignableFrom(exn.getClass()))
-                .until(() -> {
-                    testContext
-                            .getLhClient()
-                            .withDeadlineAfter(2, TimeUnit.SECONDS)
-                            .putTenant(PutTenantRequest.newBuilder()
-                                    .setId(testContext.getConfig().getTenantId().getId())
-                                    .build());
-                    return true;
-                });
+        try {
+            testContext.getLhClient().getTenant(testContext.getConfig().getTenantId());
+        } catch (StatusRuntimeException sre) {
```

Comment:
this catch is not needed, you can use Awaitility

Reply:
Fixing...